crates/voicevox_core_python_api/src/lib.rs

```diff
@@ -262,6 +264,35 @@ fn _to_zenkaku(text: &str) -> PyResult<String> {
     Ok(voicevox_core::__internal::to_zenkaku(text))
 }
 
+/// 16bit PCMにヘッダを付加しWAVフォーマットのバイナリを生成する。
+///
+/// Parameters
+/// ----------
+/// pcm : bytes
+///     16bit PCMで表現された音声データ
+/// output_sampling_rate: int
+///     pcmのサンプリングレート
+/// output_stereo: bool
+///     pcmがステレオかどうか
+///
+/// Returns
+/// -------
+/// bytes
+///     WAVフォーマットで表現された音声データ
+#[pyfunction]
+fn wav_from_s16le(
+    pcm: &[u8],
+    output_sampling_rate: u32,
+    output_stereo: bool,
+    py: Python<'_>,
+) -> PyObject {
+    PyBytes::new(
+        py,
+        &voicevox_core::blocking::wav_from_s16le(pcm, output_sampling_rate, output_stereo),
+    )
+    .into()
+}
```

Comment:
`PyObject`にしなくてもこれでいいかと。

```suggestion
fn wav_from_s16le<'py>(
    pcm: &[u8],
    output_sampling_rate: u32,
    output_stereo: bool,
    py: Python<'py>,
) -> &'py PyBytes {
    PyBytes::new(
        py,
        &voicevox_core::blocking::wav_from_s16le(pcm, output_sampling_rate, output_stereo),
    )
}
```